storage-node/src/commands/operator/set-metadata.ts

```diff
@@ -45,16 +57,55 @@ export default class OperatorSetMetadata extends ApiCommandBase {
 
   async run(): Promise<void> {
     const { flags } = this.parse(OperatorSetMetadata)
-    const { workerId, bucketId, jsonFile, endpoint } = flags
+    const { workerId, bucketId, jsonFile, endpoint, operationalStatus: statusType } = flags
+
+    let operationalStatus: INodeOperationalStatusMetadata = {}
+    switch (statusType) {
+      case 'Normal': {
+        operationalStatus = { status: NodeOperationalStatusMetadata.OperationalStatus.NORMAL }
+        break
+      }
+      case 'NoService': {
+        operationalStatus = { status: NodeOperationalStatusMetadata.OperationalStatus.NO_SERVICE }
+        break
+      }
+      case 'NoServiceFrom': {
+        operationalStatus = {
+          status: NodeOperationalStatusMetadata.OperationalStatus.NO_SERVICE,
+          noServiceFrom: (await this.datePrompt({ message: 'Enter No Service period start date' })).toISOString(),
+        }
+        break
+      }
+      case 'NoServiceDuring': {
+        operationalStatus = {
+          status: NodeOperationalStatusMetadata.OperationalStatus.NO_SERVICE,
+          noServiceFrom: (await this.datePrompt({ message: 'Enter No Service period start date' })).toISOString(),
+          noServiceTo: (await this.datePrompt({ message: 'Enter No Service period end date' })).toISOString(),
+        }
+      }
+    }
 
     const validation = new ValidationService()
-    const metadata: IStorageBucketOperatorMetadata = jsonFile
-      ? validation.validate('OperatorMetadata', JSON.parse(fs.readFileSync(jsonFile).toString()))
-      : { endpoint }
+    let metadata: IStorageBucketOperatorMetadata
+    if (jsonFile) {
+      const input = validation.validate('OperatorMetadata', JSON.parse(fs.readFileSync(jsonFile).toString()))
+      metadata = {
+        ...input,
+        operationalStatus: {
+          ...input.operationalStatus,
+          status:
+            input.operationalStatus?.status === 'Normal'
+              ? NodeOperationalStatusMetadata.OperationalStatus.NORMAL
+              : NodeOperationalStatusMetadata.OperationalStatus.NO_SERVICE,
+        },
+      }
+    } else {
+      metadata = { endpoint, operationalStatus }
+    }
 
     const encodedMetadata = '0x' + Buffer.from(StorageBucketOperatorMetadata.encode(metadata).finish()).toString('hex')
 
-    logger.info('Setting the storage operator metadata...')
+    logger.info('Se tting the storage node operational status...')
```

Comment:
```suggestion
    logger.info('Setting the storage node operational status...')
```

Reply:
Also, I think this still should be the old string since the command is also used for regular metadata?